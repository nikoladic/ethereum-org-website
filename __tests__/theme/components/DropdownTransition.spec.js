import { mount } from '@vue/test-utils'
import DropdownTransition from "@/theme/components/DropdownTransition.vue"


test('renders correctly', () => {
  const wrapper = mount(DropdownTransition, {
    slots: {
      default: '<div class="fake-msg"></div>'
    }
  });
  expect(wrapper.element).toMatchSnapshot()
});