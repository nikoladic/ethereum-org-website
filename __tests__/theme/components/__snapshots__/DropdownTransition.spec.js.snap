// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="fake-msg"
  name="dropdown"
/>
`;
